src/hooks/use-fetch-posts/use-fetch-posts.js

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+export const fetchPosts = async (
+  subreddit,
+  postAmount = 500,
+  lastPostId,
+  fetchedPosts,
+) => {
+  let postId = typeof (lastPostId) === 'undefined' ? null : lastPostId;
+  let posts = typeof (fetchedPosts) === 'undefined' ? [] : fetchedPosts;
+  let noMorePosts = false;
```

Comment:
No need to use `let` here. You can simply define it as a `const` a few lines down